cpp/benchmarks/string/copy.cu

```diff
@@ -24,7 +24,8 @@
 #include <cudf_test/column_wrapper.hpp>
 
 #include <algorithm>
-#include <random>
+
+#include <thrust/shuffle.h>
```

Comment:
Nit: Move the `include` statement below to be above all other include statements.

Reply:
updated.